src/core/gltf2exporter/gltf2exporter_p.cpp

```diff
@@ -60,6 +60,7 @@ QString generateUniqueFilename(const QDir &dir, QString filename)
     int num = 1;
     while (dir.exists(filename)) {
         filename = QStringLiteral("%1-%2.%3").arg(basename).arg(num).arg(ext);
+        num++;
```

Comment:
++num even since you don't need to read back the value immediately :)

Reply:
done